sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeEntitiesWithResolutions.md

```diff
@@ -0,0 +1,172 @@
+# Resolve entities to standard formats with NER resolutions
+
+This sample demonstrates how to resolve entities in a document by using the named entity recognition (NER) feature of the Azure Cognitive Service for Language. NER resolutions provide standard, predictable formats for common types and concepts (such as dates, quantities, and dimensions) to help you retrieve and process information from documents more efficiently. To learn more about NER resolutions, see [here][NER_Resolutions].
+
+## Create a `TextAnalyticsClient`
+
+To create a new `TextAnalyticsClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create simply with an API key.
```

Comment:
It's generally advised to leave opinionated complexity like "simply" out. Not everyone may find it so simple to find/generate an API key.
```suggestion
To create a new `TextAnalyticsClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create with an API key.
```